### ISSUE TITLE
Dolly example no longer works ...

### ISSUE THREAD

**Issue by rcherukuri12** [chengchingwen/Transformers.jl#156]
Looks like this package ..is no longer current and doesn't work with LLMs 

textenc = hgf"databricks/dolly-v2-12b:tokenizer"
Warning: fuse_unk is unsupported, the tokenization result might be slightly different in some cases

julia> model = todevice(hgf"databricks/dolly-v2-12b:ForCausalLM")
ERROR: unknown type: torch.BFloat16Storage

**Comment by chengchingwen** [comment]
Which version of Transformers.jl and Pickle.jl?

**Comment by rcherukuri12** [comment]
(@v1.9) pkg> status Transformers
Status `~/.julia/environments/v1.9/Project.toml`
  [21ca0261] Transformers v0.2.8

**Comment by chengchingwen** [comment]
Ok, so you would need to update Pickle.jl to 0.3.3 which adds support for bfloat16.

**Comment by rcherukuri12** [comment]
pkg>update Pickle
 wouldn't do it.

**Comment by rcherukuri12** [comment]
on linux / ubuntu ..is this platform specific ?

**Comment by chengchingwen** [comment]
Not really. What happens if you explicitly `add Pickle@0.3.3`? I'm guessing there are some compat issues that block the update.

**Comment by rcherukuri12** [comment]
(@v1.9) pkg> add Pickle@0.3.3
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package ReinforcementLearningZoo [d607f57d]:

**Comment by rcherukuri12** [comment]
I will create a local project and activate and see if it works ..

**Comment by rcherukuri12** [comment]
Similar warning as before ... 
textenc = hgf"databricks/dolly-v2-12b:tokenizer"
┌ Warning: fuse_unk is unsupported, the tokenization result might be slightly different in some cases.
└ @ Transformers.HuggingFace ~/.julia/packages/Transformers/lD5nW/src/huggingface/tokenizer/utils.jl:42

will load the model and see if it works.

**Comment by rcherukuri12** [comment]
This time ...no error , but model hangs ..
model = todevice(hgf"databricks/dolly-v2-12b":ForCausalLM")

**Comment by chengchingwen** [comment]
The warning can usually be ignored

> model = todevice(hgf"databricks/dolly-v2-12b":ForCausalLM")

That is a big model, which takes time to be moved to GPU. And there is an extra `"`.

**Comment by rcherukuri12** [comment]
I have a 3080 ti ..will wait to see and let you know . Thank you

**Comment by rcherukuri12** [comment]
finally it put out an error:
ERROR: LoadError: syntax: cannot juxtapose string literal

**Comment by chengchingwen** [comment]
That is because the extra `"`

**Comment by rcherukuri12** [comment]
Let me see where it is happening ... it is this line:
model = todevice(hgf"databricks/dolly-v2-12b:ForCausalLM")

**Comment by rcherukuri12** [comment]
The other thing , I noticed ..nvidia-smi was constant ...almost telling it was not copying there ..

**Comment by rcherukuri12** [comment]
I will copy into a vim editor and try it out ..that way , double quotes sometimes comes up with special chars...

**Comment by rcherukuri12** [comment]
Still no luck. It just kills my shell after a while . Will come back and try later .
For now I will stick to just using OpenAI.jl and continue my work.
Thank you for trying to help.

**Comment by chengchingwen** [comment]
It sounds like the process might be killed due to OOM.
